src/plugins/intel_gna/src/common/numerical_utils.hpp

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <algorithm>
+#include <cmath>
+#include <limits>
+#include <type_traits>
+
+namespace ov {
+namespace intel_gna {
+namespace common {
+
+template <typename T>
+inline T float_to_integer(float a) {
+    return static_cast<T>((a < 0.0f) ? (a - 0.5f) : (a + 0.5f));
+}
+
+inline int8_t float_to_int8(float a) {
+    return float_to_integer<int8_t>(a);
+}
+inline int16_t float_to_int16(float a) {
+    return float_to_integer<int16_t>(a);
+}
+inline int32_t float_to_int32(float a) {
+    return float_to_integer<int32_t>(a);
+}
+inline int64_t float_to_int64(float a) {
+    return float_to_integer<int64_t>(a);
+}
+
+/**
+ * @brief Compare two float values and return true if they are equal with given accuracy
+ * @param p1 First floating point value
+ * @param p2 Second floating point value
+ * @param accuracy accuracy of comparision
+ * @return Returns true if two floating point values are equal
+ */
+template <typename T, typename std::enable_if<std::is_floating_point<T>::value, int>::type = 0>
+bool is_fp_eq(T p1, T p2, T accuracy = std::numeric_limits<T>::epsilon()) {
```

Comment:
If you really need to change the name of the function, I'd suggest are_fq_eq, because we're comparing two numbers, not one. And also I'm not sure what is the current naming style preference. It's changing to fast to follow... ;-)

Reply:
I will change it to `are_fq_eq`.

I followed where [openvino code style](https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/coding_style.md) where is mentioned that method and functions should be `snake_case` ( the rule is also present in linked  ncc configuration file)